data/custom.xml

```diff
@@ -55,7 +55,7 @@
 
     <magic priority="50">
       <match value="PK\003\004" type="string" offset="0">
-        <match value="[Content_Types].xml" type="string" offset="30">
+        <match value="[Content_Types].xml" type="string" offset="30:4096">
```

Comment:
this would allow marcel to look up wider range (I have to admit it's heuristic though) for `[Content_Types].xml`.

I'm guessing it can be intentionally done for low cost operation, but let me know what you think.